apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifimanager_main.c

```diff
@@ -444,12 +418,9 @@ static void utc_wifimanager_get_config_n(void)
 	wifi_manager_ap_config_s config;
 	ret = wifi_manager_get_config(&config);
 	if (ret == WIFI_MANAGER_SUCCESS) {
-		printf("====================================\n");
-		printf("SSID: %s\n", config.ssid);
-		printf("SECURITY TYPE: %d\n", config.ap_auth_type);
-		printf("====================================\n");
-		res = strncmp(config.ssid, CONFIG_EXAMPLES_TESTCASE_WIFI_MANAGER_UTC_SSID, strlen(config.ssid));
-		res = strncmp(config.passphrase, CONFIG_EXAMPLES_TESTCASE_WIFI_MANAGER_UTC_PASSPHRASE, strlen(config.passphrase));
+		_print_ap_config(&config);
+		res = strncmp(config.ssid, g_ssid, strlen(config.ssid));
+		res = strncmp(config.passphrase, g_passphrase, strlen(config.passphrase));
```

Comment:
Same as above. Please check the result of line 422 first.

Reply:
Thanks I'll modify it